packages/atomic/README.md

```diff
@@ -14,7 +14,7 @@ Once you have cloned the repo, follow the instructions in the top-level [README.
 To start the project in development mode, run:
 
 ```bash
-npm run dev
+npm run start
```

Comment:
This was out of date as the `dev` script doesn't exist in `package.json` anymore